addons/point_of_sale/static/src/js/Popups/ClosePosPopup.js

```diff
@@ -14,44 +14,14 @@ odoo.define('point_of_sale.ClosePosPopup', function(require) {
         constructor() {
             super(...arguments);
             this.manualInputCashCount = false;
-            this.cashControl = this.env.pos.config.cash_control;
             this.moneyDetailsRef = useRef('moneyDetails');
             this.closeSessionClicked = false;
             this.moneyDetails = null;
             this.state = useState({});
+            Object.assign(this, this.props.info);
         }
         async willStart() {
```

Comment:
One more thing. Can you put comment so that no one will accidentally remove it in stable?
```
/**
 * @deprecated Don't remove. There might be overrides.
 */
```

Then we remove this in master. Note that in saas-15.3, it's called onWillStart.